packages/functions/src/addcachetag/index.ts

```diff
@@ -3,7 +3,7 @@ import { getContext } from '../get-context';
 export const addCacheTag = (tag: string | string[]) => {
   const api = getContext().addCacheTag;
   if (api) {
-    return api.addCacheTag(tag);
+    return api(tag);
```

Comment:
```suggestion
const addCacheTag = getContext().addCacheTag;
if (addCacheTag) {
    return addCacheTag(tag);
```

I think this reads better